easybuild/easyconfigs/s/STRUMPACK/STRUMPACK-6.1.0-foss-2020b.eb

```
@@ -0,0 +1,52 @@
+easyblock = 'CMakeMake'
+
+name = 'STRUMPACK'
+version = '6.1.0'
+
+homepage = 'https://fastmath-scidac.llnl.gov/software/strumpack.html'
+description = """STRUMPACK - STRUctured Matrix PACKage - Fast linear solvers and preconditioner
+ for both dense and sparse systems using low-rank structured factorization with randomized sampling."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'pic': True}
```

The mpi compilers can be automatically enabled with `usempi`. Then you can remove `CMAKE_CXX_COMPILER`, `CMAKE_C_COMPILER` and `CMAKE_Fortran_COMPILER` from `configopts`:
```suggestion
toolchainopts = {'pic': True, 'usempi': True}
```